marimo/_cli/sandbox.py

```diff
@@ -140,7 +146,14 @@ def run_in_sandbox(
         "--no-project",
         "--with-requirements",
         temp_file_path,
-    ] + cmd
+    ]
+
+    marimo_help_cmd = uv_run + ["marimo", "edit", "--help"]
+    p = subprocess.run(marimo_help_cmd, capture_output=True)
+    if "--skip-sandbox-prompt" in p.stdout.decode():
+        cmd += ["--skip-sandbox-prompt"]
+
+    cmd = uv_run + cmd
```

Comment:
This is the best I could come up with to check if the version of marimo that uv will load has the `--skip-sandbox-prompt`. It does mean that we create an isolated environment twice, however ... @mscolnick let me know if you have other ideas